src/functions.rs

```diff
@@ -2393,13 +2428,206 @@ impl TryFrom<&ValidatedStatement<'_>> for sexp::Sexp {
             ValidatedStatement::ResourcetransRule(r) => Ok(Sexp::from(r)),
             ValidatedStatement::FscRule(fs) => Sexp::try_from(fs),
             ValidatedStatement::DomtransRule(d) => Ok(Sexp::from(d)),
+            ValidatedStatement::Deferred(d) => Ok(Sexp::from(d)),
             // Sids in functions should error during validation.  Global SIDs are filtered out
             // before sexp generation
             ValidatedStatement::Sid(_) => Err(InternalError::new().into()),
         }
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord)]
+pub enum DeferredStatement {
+    Call(DeferredCall),
+    Validation, // TODO
+}
+
+impl DeferredStatement {
+    // Copy into the parent context, converting argument names
+    fn parent_copy(&self, caller_args: &Vec<CilArg>, orig_function: &FunctionInfo) -> Self {
+        match self {
+            // TODO: 0
```

Comment:
0?

Reply:
You don't see any '0' because I did the TODO :)